dedicated-examples/mock-camera-microphone-access/README.md

```diff
@@ -1 +1,29 @@
-# mock camera/microphone access
+# Mock Camera/Microphone Access
+
+This example shows how to mock data for microphone and camera. This example runs in **Chrome**.
+
+**Test Code**: [mock-camera-microphone-access.js](mock-camera-microphone-access.js)
+
+The test page requests a permission to record audio and video. Resulting streams are directed to corresponding `<audio>` and `<video>` elements on the page. When the `<button>` element registers a click, a snapshot of the video stream draws on the `<img>` element if video stream is present.
+
+The first test function uses [Selector.addCustomDOMProperties](https://devexpress.github.io/testcafe/documentation/reference/test-api/selector/addcustomdomproperties.html) method to read `<img>`'s `.src` property and check whether it holds an image.
+
+The second test function uses [Selector.addCustomDOMProperties](https://devexpress.github.io/testcafe/documentation/reference/test-api/selector/addcustomdomproperties.html) method to read `<audio>`'s `.srcObject` property. If audio stream is present, it returns `true`. This value is tested with [t.expect.ok](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/expect/ok.html) method.
+
+`getUserMedia` allows access to the Media Stream API. When called, it spawns a dialog window. Use `--use-fake-ui-for-media-stream` Chrome flag to grant the permission without user input.
```

Comment:
- "When called" When what is called? 
- "it spawns"? Why do you use this verb?
- "to grant the permission" Why do you use an article for permission?